src/screens/LoginPage/LoginPage.tsx

```diff
@@ -53,17 +54,14 @@ function LoginPage(): JSX.Element {
                 }}
                 required
               />
-              <input
-                type="submit"
-                className="btn"
-                value="Login"
-                onClick={login_link}
-              />
+              <button type="submit" className={styles.loginbtn} value="Login">
```

Comment:
see the new code it should go to homepage on clicking

Reply:
@yasharth291 again you worked on this part so I'll check

Reply:
it was in the new code merged

Reply:
seen, please verify at your end

Reply:
![Capture](https://user-images.githubusercontent.com/55004621/122251446-21d0d200-cee8-11eb-9851-61f7bdcc2a0e.PNG)

see this

Reply:
you have to make the button clickable